### ISSUE TITLE
[NVIM v0.6.0-dev] Diaglist does not let go of cursor focus and makes nvim unusable

### ISSUE THREAD

**Issue by fnune** [onsails/diaglist.nvim#7]
```
$ nvim --version
NVIM v0.6.0-dev+274-g6188926e0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

After installing diaglist, here's what happens when I enter a buffer with an LSP attached to it:

https://user-images.githubusercontent.com/16181067/137697948-ade35c34-8e3f-47ef-8c5d-4327a2661e76.mp4

Basically, the diaglist window appears and steals focus. If I try to go to another buffer, focus immediately goes back to diaglist.

I've set up diaglist with just:

```
require("diaglist").init({})
```

---

On NVIM v0.5.1, diaglist works just fine!

**Comment by rockneverdies55** [comment]
Using the released version 0.6 (not dev) and still the same issue.

**Comment by onsails** [comment]
I've upgraded my neovim to 0.6.0 and reproduced the problem. Working on fix.
